src/node_buffer.cc

```diff
@@ -541,6 +615,78 @@ void WriteDoubleBE(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+template <typename T, enum Endianness endianness,
+         T (*strtoT)(const char*, char**, int), T min, T max>
+uint32_t WriteInt64Generic(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK(args[0]->IsObject());
+  CHECK(args[1]->IsString() || args[1]->IsNumber());
+  ARGS_THIS(args[0].As<Object>())
```

Comment:
You can remove the `IsObject()` above because that's already checked by `HasInstance()` if you pass a `Local<Value>`.